nodeList.json

```diff
@@ -0,0 +1,900 @@
+[
```

Comment:
I ran the Schema and the Node List via http://www.jsonschemavalidator.net/ and it checks out. In fact Javascript and above site errors out parsing { instrad of [